mRemoteV1/Connection/Protocol/RDP/Connection.Protocol.RDP.cs

```diff
@@ -547,6 +547,7 @@ private void SetRedirection()
 				_rdpClient.AdvancedSettings2.RedirectPrinters = _connectionInfo.RedirectPrinters;
 				_rdpClient.AdvancedSettings2.RedirectSmartCards = _connectionInfo.RedirectSmartCards;
 				_rdpClient.SecuredSettings2.AudioRedirectionMode = (int)_connectionInfo.RedirectSound;
+                _rdpClient.AdvancedSettings8.AudioCaptureRedirectionMode = ((int)_connectionInfo.RedirectAudioInput) != 0;
```

Comment:
_rdpClient.AdvancedSettings8.AudioCaptureRedirectionMode = Convert.ToBoolean(_connectionInfo.RedirectMicrophones);

Looks a little cleaner... But at a minimum you don't need the cast or parenthesis.